src/Components/CreateImageWizard/steps/Repositories/Repositories.tsx

```diff
@@ -389,207 +400,387 @@ const Repositories = () => {
-                      </Td>
-                    </Tr>
-                  );
-                })}
-              </Tbody>
-            </Table>
-          )}
-        </PanelMain>
-      </Panel>
-      <Pagination
-        itemCount={count ?? PAGINATION_COUNT}
-        perPage={perPage}
-        page={page}
-        onSetPage={(_, newPage) => setPage(newPage)}
-        onPerPageSelect={handlePerPageSelect}
-        variant={PaginationVariant.bottom}
-      />
-    </Grid>
-  );
+                        <Td dataLabel={'Name'}>
+                          {name}
+                          {origin === ContentOrigin.UPLOAD ? (
+                            <UploadRepositoryLabel />
+                          ) : (
+                            <>
+                              <br />
+                              <Button
+                                component="a"
+                                target="_blank"
+                                variant="link"
+                                icon={<ExternalLinkAltIcon />}
+                                iconPosition="right"
+                                isInline
+                                href={url}
+                              >
+                                {url}
+                              </Button>
+                            </>
+                          )}
+                        </Td>
+                        <Td dataLabel={'Architecture'}>
+                          {distribution_arch || '-'}
+                        </Td>
+                        <Td dataLabel={'Version'}>
+                          {distribution_versions || '-'}
+                        </Td>
+                        <Td dataLabel={'Packages'}>{package_count || '-'}</Td>
+                        <Td dataLabel={'Status'}>
+                          <RepositoriesStatus
+                            repoStatus={status || 'Unavailable'}
+                            repoUrl={url}
+                            repoIntrospections={last_introspection_time}
+                            repoFailCount={failed_introspections_count}
+                          />
+                        </Td>
+                      </Tr>
+                    );
+                  })}
+                </Tbody>
+              </Table>
+            )}
+          </PanelMain>
+        </Panel>
+        <Pagination
+          itemCount={count ?? PAGINATION_COUNT}
+          perPage={perPage}
+          page={page}
+          onSetPage={(_, newPage) => setPage(newPage)}
+          onPerPageSelect={handlePerPageSelect}
+          variant={PaginationVariant.bottom}
+        />
+      </Grid>
+    );
+  } else {
+    return (
+      <>
+        <Alert
+          variant="info"
+          title="The repositories seen below are from the selected content template and have been added automatically. 
+              If you do not want these repositories in your image, you can modify the content template or choose another snapshot option."
```

Comment:
link here inside this text that would redirect the user right to the edit modal of the template in question could be a nice addition ✨ 
then the user wouldn't have to backtrack in the wizard to see the selected template and manually navigate to the templates tab, where he would have to find it and click edit in the kebab to get to the same place 😄

Reply:
oh yea true! added that in